jetbrains: 2025.2.1 -> 2025.3.2

jamesward opened the pull request:
jetbrains: 2025.2.1 -> 2025.3.2
    
    jetbrains.clion: 2025.2.5 -> 2025.3.1
    jetbrains.datagrip: 2025.3 -> 2025.3.2
    jetbrains.dataspell: 2025.2.3 -> 2025.3.1
    jetbrains.gateway: 2025.2.5 -> 2025.3
    jetbrains.goland: 2025.2.5 -> 2025.3
    jetbrains.idea: 2025.2.5 -> 2025.3.1
    jetbrains.mps: 2025.2.1 -> 2025.3
    jetbrains.phpstorm: 2025.2.5 -> 2025.3.1
    jetbrains.pycharm: 2025.2.5 -> 2025.3.1
    jetbrains.rider: 2025.3.0.3 -> 2025.3.1
    jetbrains.ruby-mine: 2025.2.5 -> 2025.3.1
    jetbrains.rust-rover: 2025.2.5 -> 2025.3.1
    jetbrains.webstorm: 2025.2.5 -> 2025.3.1

## Things done

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

aoli-al commented:
Yes, I can launch idea-ultimate with jbr 21.0.9. I tired to enable AI assistant but I don't have a subscription so it just shows the front page.

aoli-al commented:
https://github.com/NixOS/nixpkgs/pull/470840 is merged. @jamesward can you please rebase and check if AI Assistant is working?

jamesward committed:
jetbrains: 2025.2.1 -> 2025.3.2

jetbrains.clion: 2025.2.5 -> 2025.3.1
jetbrains.datagrip: 2025.3 -> 2025.3.2
jetbrains.dataspell: 2025.2.3 -> 2025.3.1
jetbrains.gateway: 2025.2.5 -> 2025.3
jetbrains.goland: 2025.2.5 -> 2025.3
jetbrains.idea: 2025.2.5 -> 2025.3.1
jetbrains.mps: 2025.2.1 -> 2025.3
jetbrains.phpstorm: 2025.2.5 -> 2025.3.1
jetbrains.pycharm: 2025.2.5 -> 2025.3.1
jetbrains.rider: 2025.3.0.3 -> 2025.3.1
jetbrains.ruby-mine: 2025.2.5 -> 2025.3.1
jetbrains.rust-rover: 2025.2.5 -> 2025.3.1
jetbrains.webstorm: 2025.2.5 -> 2025.3.1

jamesward commented:
AFAICT the JCEF stuff is working now.

theCapypara commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 469024`
Commit: [`53e910249bca0856b71b1436840842f7a9874966`](https://github.com/NixOS/nixpkgs/commit/53e910249bca0856b71b1436840842f7a9874966) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/53e910249bca0856b71b1436840842f7a9874966..pull/469024/head))
Merge: [`3224a45ff3bb9447455520a6f6b97281796be911`](https://github.com/NixOS/nixpkgs/commit/3224a45ff3bb9447455520a6f6b97281796be911)

Logs: https://github.com/theCapypara/nixpkgs-review-gha/actions/runs/20569817071


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea</li>
    <li>jetbrains.mps</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea</li>
    <li>jetbrains.mps</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea</li>
    <li>jetbrains.mps</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea</li>
    <li>jetbrains.mps</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

theCapypara reviewed (approved):
I also tested the binaries locally. I don't really have a good way to test the JCEF things though.
